tests/e2e/specs/wordpress/quickEdit.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * WordPress dependencies
+ */
+import {
+  loginUser,
+  switchUserToAdmin,
+  visitAdminPage,
+} from '@wordpress/e2e-test-utils';
+/**
+ * Internal dependencies
+ */
+import { createNewStory } from '../../utils';
+
+describe('Quick Edit', () => {
+  beforeAll(async () => {
+    await loginUser('author', 'password');
+  });
+
+  afterAll(async () => {
+    await switchUserToAdmin();
+  });
+
+  it('save quick edit', async () => {
```

Comment:
More accurate would be something like "it should save story without breaking markup"

Reply:
Fixed